internal/repository/repository.go

```diff
@@ -19,9 +19,9 @@ import (
 	"github.com/restic/restic/internal/pack"
 	"github.com/restic/restic/internal/restic"
 	"github.com/restic/restic/internal/ui/progress"
+	"golang.org/x/sync/errgroup"
```

Comment:
I would leave this include grouped under the "non-restic" includes...

Reply:
Is moved back.